cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -149,34 +140,20 @@ def test_provider_summary_scvmm():
 @pytest.mark.tier(2)
 def test_host_info_scvmm():
     """
-    The purpose of this test is to verify that SCVMM-SP1 hosts are not
-    only added, but that the host information details are correct.  Take
-    the time to spot check at least one host.
+    Testing that host info is correct in CFME
 
```

Comment:
I can see mistake in this sentence here, as I went through your steps which are very clear; This sentence should be like
``` Testing the host info is correct in CFME```(if I am not wrong - current sentence indicates that you are talking about specific host)

Reply:
Good catch, I forgot _the_ article! I'll make it read, `Testing to make sure that the host info is correct in CFME`